R/plan.R

```diff
@@ -1,19 +1,48 @@
-plan <- drake::drake_plan(
-  import_metadata = get_data(config::get("metadata")$synID,
-                config::get("metadata")$version),
-  import_counts = get_data(config::get("counts")$synID,
-                    config::get("counts")$version),
-  counts = tibble::column_to_rownames(import_counts, var = config::get("counts")$`gene id`),
-  clean_md = clean_covariates(md = import_metadata, factors = config::get("factors"),
-                              continuous = config::get("continuous")),
-  #covar_correlation = CovariateAnalysis::getAssociationStatistics(clean_md, PVAL = 0.05),
-  biomart_results = get_biomart(count_df = counts,
-                                gene_id = config::get("biomart")$`gene id`,
-                                synid = config::get("biomart")$synID,
-                                version = config::get("biomart")$version,
-                                filters = config::get("biomart")$filters,
-                                host = config::get("biomart")$host,
-                                organism = config::get("biomart")$organism),
-  filtered_counts = filter_genes(md = clean_md, count_df = counts),
-  cqn_counts = cqn(filtered_counts, biomart_results)
+#' Execute the drake RNA-seq plan
+#'
+#' This function wraps the \code{"drake::plan()"}.
+#'
+#' @param metadata_id Synapse ID to clean metadata file with sample identifiers in a
+#' column and variables of interest as column names. There cannot be any missing values.
+#' @param metadata_version Optionally, include Synapse file version number. If omitted,
+#' current version will be downloaded.
+#' @param counts_id Synapse ID to counts data frame with identifiers to the metadata as
+#' column names and gene ids in a column.
+#' @param counts_version Optionally, include Synapse file version number.
+#' @param gene_id_input Column name of the gene ids in the counts_id file.
+#' @param factor_input Vector of factor variables. Variables must be present
+#' in the metadata as column names.
+#' @param continuous_input Vector of continuous variables. Variables must be
+#' present in the metadata as column names.
+#' @param biomart_id Synapse ID to biomart object.
+#' @param biomart_version Optionally, include Synapse file version number.
+#' @inheritParams get_biomart
+#' @export
+rnaseq_plan <- function(metadata_id, metadata_version, counts_id,
```

Comment:
I think you need to run `devtools::document()` to make sure the .Rd documentation gets generated.